Sources/HiPMobile/ServiceAccessLayer/FeatureToggleApiDto/FeatureToggleApiAccess.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (C) 2017 History in Paderborn App - Universität Paderborn
```

Comment:
This file seems to be in the wrong folder. It should probably be in FeatureToggleApiAccess like the corresponding interface.